app/src/main/java/com/ddoczi/tasky/agenda/presentation/detail/agenda/AgendaDetailEvent.kt

```diff
@@ -1,21 +1,22 @@
 package com.ddoczi.tasky.agenda.presentation.detail.agenda
 
 import java.time.LocalDate
-import java.time.LocalDateTime
 import java.time.LocalTime
 
 sealed class AgendaDetailEvent {
     data object OnEdit : AgendaDetailEvent()
     data object OnSave : AgendaDetailEvent()
     data object OnClose : AgendaDetailEvent()
     data object OnDelete : AgendaDetailEvent()
-    data class OnDateSelected(val date: LocalDateTime) : AgendaDetailEvent()
-    data class OnTimeSelected(val time: LocalDateTime) : AgendaDetailEvent()
-    data class OnFromDateSelected(val date: LocalDate) : AgendaDetailEvent()
-    data class OnFromTimeSelected(val time: LocalTime) : AgendaDetailEvent()
-    data object OnNotificationClick : AgendaDetailEvent()
-    data object OnNotificationDismiss : AgendaDetailEvent()
-    data object OnNotificationSelect : AgendaDetailEvent()
+    data class OnDateSelected(val date: LocalDate) : AgendaDetailEvent()
+    data class OnTimeSelected(val time: LocalTime) : AgendaDetailEvent()
+    data class OnFromDateSelected(val fromDate: LocalDate) : AgendaDetailEvent()
+    data class OnToDateSelected(val toDate: LocalDate) : AgendaDetailEvent()
+    data class OnFromTimeSelected(val fromTime: LocalTime) : AgendaDetailEvent()
+    data class OnToTimeSelected(val toTime: LocalTime) : AgendaDetailEvent()
```

Comment:
I think these events are a bit confusing. If you were a new dev in a project and you'd see `OnTimeSelected`, `OnFromTimeSelected` and `OnToTimeSelected`, would you be able to tell the differences? 
You can get rid of the first two events here, since there's no scenario where you need all three different time events. Maybe calling one `OnTimeSelected` and the other one `OnToTimeSelected`, since tasks and reminders just have a normal time and can respond to the first event while events can also respond to the second